gallery/src/styles/Nivo/NivoBarExample.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { ResponsiveBar } from '@nivo/bar';
+import { NIVO_COLORS } from '@sonatype/react-shared-components';
+import { rawBarData } from './barData';
+
+const barData = rawBarData.map(([id, value]) => ({ id, value }));
```

Comment:
Why isn't this data just defined in the expected format in the first place?

Reply:
> Why isn't this data just defined in the expected format in the first place?

Updated data here: 6c3e8a61ff216cb785af2b0e9fcb2a80cf53b0ed